packages/data-context/src/sources/VersionsDataSource.ts

```diff
@@ -160,6 +162,47 @@ export class VersionsDataSource {
       }
     }
 
+    try {
+      const projectPath = this.ctx.currentProject
+
+      if (projectPath) {
+        const dependenciesToCheck = WIZARD_DEPENDENCIES
+
+        debug('Checking %d dependencies in project', dependenciesToCheck.length)
+        // Check all dependencies of interest in parallel
+        const dependencyResults = await Promise.allSettled(
```

Comment:
Hardly see `allSettled`, nice! I think we could have got away with `Promise.all` and `return undefined` with a filter instead of throwing an error and using `fulfilled`, but fine with this, too.